README.md

```diff
@@ -140,7 +153,9 @@ If you are familiar with the great TensorFlow Datasets, here are the main differ
 
 Similar to TensorFlow Datasets, 🤗 Datasets is a utility library that downloads and prepares public datasets. We do not host or distribute most of these datasets, vouch for their quality or fairness, or claim that you have license to use them. It is your responsibility to determine whether you have permission to use the dataset under the dataset's license.
 
-If you're a dataset owner and wish to update any part of it (description, citation, etc.), or do not want your dataset to be included in this library, please get in touch through a [GitHub issue](https://github.com/huggingface/datasets/issues/new). Thanks for your contribution to the ML community!
+Moreover 🤗 Datasets may run python code defined by the dataset authors to parse certain data formats or structures. For security reasons, we ask users to check the dataset scripts they're going to run beforehand and pin the `revision` of the repositories they use.
+
+If you're a dataset owner and wish to update any part of it (description, citation, etc.), or do not want your dataset to be included in the Hugging Face Hub, please get in touch by opening a discussion or a pull request in the Community tab of the dataset page. Thanks for your contribution to the ML community!
```

Comment:
Nit: I would also add "license" to the list, as it is missing in many datasets or it can be wrong
```suggestion
If you're a dataset owner and wish to update any part of it (description, citation, license, etc.), or do not want your dataset to be included in the Hugging Face Hub, please get in touch by opening a discussion or a pull request in the Community tab of the dataset page. Thanks for your contribution to the ML community!
```